media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
+* Support for common image formats and GIF animation
+* Supports tagging images, and filtering images by tags (IPTC)
+* Browse thumbnails recursively down a folder tree
+* Dynamic thumbnails loading
+* Image transformation and color manipulation
+* Display image information and metadata
+* Does not depend on any desktop environment."
+HOMEPAGE="https://github.com/oferkv/phototonic"
+COPYRIGHT="2015-2018 Over Kashayov
+	2015-2018 Christopher roy
+	2015-2018 Krzysztof Pyrkosz
+	2015-2018 Roman Chistokhodov
+	2015-2018 Thomas Lübking
+	2015-2018 Tung Le"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/oferkv/phototonic/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a8b2dbc81750efabb65a4732b1704641afd4a4f772ef9c90817871ca2a52f5ef"
+SOURCE_FILENAME="phototonic-$portVersion.tar.gz"
+PATCHES="phototonic-$portVersion.patchset"
+ADDITIONAL_FILES="phototonic.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+USER_SETTINGS_FILES="
+	settings/PhotoTonic directory
+	"
+PROVIDES="
+	phototonic$secondaryArchSuffix = $portVersion
+	app:PhotoTonic = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexiv2$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexiv2$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:qmake >= 5
```

Comment:
```diff
-	cmd:qmake >= 5
+	cmd:qmake$secondaryArchSuffix >= 5
```
(That's because qt5 provides `cmd:qmake$secondaryArchSuffix` instead of `cmd:qmake`.)
https://github.com/haikuports/haikuports/blob/1f4d54498766f834a475eb6ad34a06c02b2a9d74/dev-qt/qt5/qt5-5.11.2.recipe#L266

Reply:
ok